### ISSUE TITLE
angular 12 does not compile with 3.0.0-beta.0 

### ISSUE THREAD

**Issue by StefanSchoof** [eclipsesource/jsonforms#1867]
### Describe the bug

Having a angular 12 app and using 3.0.0-alpha.3 works:
```
ng build

✔ Browser application bundle generation complete.
✔ Copying assets complete.
✔ Index html generation complete.

Initial Chunk Files               | Names         |      Size
main.00656aed06aad56e9caa.js      | main          |   1.30 MB
styles.06aefdba6bcc06d5b416.css   | styles        |  71.87 kB
polyfills.1b0e428dabdd636440c3.js | polyfills     |  36.22 kB
runtime.e544e5ce94d125027ace.js   | runtime       |   3.03 kB

                                  | Initial Total |   1.40 MB

Lazy Chunk Files                  | Names         |      Size
107.adc73bb5ec1dfd5e1bcd.js       | -             | 685.66 kB
299.cc297a3b5f3532ccddbc.js       | -             | 221.05 kB
320.d850b0c2edf226b61632.js       | -             |  16.46 kB
457.5d0a24a2af3142780f1e.js       | -             |  12.74 kB
894.454f6f08ab10e9483de2.js       | -             |  10.58 kB
941.856c94e25d5f50cd142a.js       | -             |   8.95 kB
257.9593493d79e7d170b523.js       | -             |   2.13 kB
common.d0741b7e90f4e7fa3a4f.js    | common        |   1.40 kB

Build at: 2022-01-11T09:25:42.913Z - Hash: c8b96dd1724c3ea4ce50 - Time: 47777ms

Warning: <snip>.module.ts depends on '@jsonforms/angular'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>.module.ts depends on '@jsonforms/angular-material'. CommonJS or AMD 
dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
```

After update to beta.0
```
> ng build

⠋ Generating browser application bundles (phase: setup)...
Compiling @jsonforms/angular : main as commonjs

Compiling @jsonforms/angular-material : module as esm5
✔ Browser application bundle generation complete.

Warning: <snip>\node_modules\@jsonforms\angular-material\__ivy_ngcc__\lib\esm\other\master-detail\master.js depends on 'lodash/get'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\angular-material\__ivy_ngcc__\lib\esm\other\master-detail\master.js depends on 'lodash/some'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\angular-material\__ivy_ngcc__\lib\esm\other\object.renderer.js depends on 'lodash/isEmpty'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\angular-material\__ivy_ngcc__\lib\esm\other\object.renderer.js depends on 'lodash/startCase'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>.module.ts depends on '@jsonforms/angular'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies



./node_modules/@jsonforms/angular-material/__ivy_ngcc__/lib/esm/other/index.js:28:0-33 - Error: The requested module './table.renderer' contains conflicting star exports for the name 'ɵ0' with the previous requested module './label.renderer'
```

### Expected behavior

Updating from alpha-3 to beta-0 does not beak the build

### Steps to reproduce the issue

1. Having a angular 12 app with jsonforms 3.0.0-alpha.3
2. `npm update @jsonforms/angular @jsonforms/angular-material @jsonforms/core --force` (the force is needed because angular 12 and json forms does use different zonejs versions)
3. ng build

### In which browser are you experiencing the issue?

n/a

### Framework

Core, Angular

### RendererSet

Material

**Comment by sdirix** [comment]
Thank you for the report. We'll take a look!

**Comment by sdirix** [comment]
@StefanSchoof This fix is part of the `3.0.0-beta.1` release. Can you verify that the issue is fixed for you?

**Comment by StefanSchoof** [comment]
Thanks, the beta.1 is working.

With the updates the CommonJS warning have changed. Now it looks like only like dependency lodash an no longer jsonforms itself is generating this warning.

```
Warning: <snip>\node_modules\@jsonforms\angular-material\__ivy_ngcc__\lib\esm\controls\numbenderer.js depends on 'lodash/merge'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\angular-material\__ivy_ngcc__\lib\esm\other\master-il\master.js depends on 'lodash/get'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\angular-material\__ivy_ngcc__\lib\esm\other\master-il\master.js depends on 'lodash/some'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\angular-material\__ivy_ngcc__\lib\esm\other\object.erer.js depends on 'lodash/isEmpty'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\angular-material\__ivy_ngcc__\lib\esm\other\object.erer.js depends on 'lodash/startCase'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\angular\__ivy_ngcc__\lib\esm\jsonforms-root.compones depends on 'ajv'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\angular\__ivy_ngcc__\lib\esm\jsonforms.component.jsends on 'lodash/maxBy'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\core\lib\jsonforms-core.esm.js depends on 'ajv-form. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\core\lib\jsonforms-core.esm.js depends on 'lodash/cDeep'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\core\lib\jsonforms-core.esm.js depends on 'lodash/eith'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\core\lib\jsonforms-core.esm.js depends on 'lodash/fr'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\core\lib\jsonforms-core.esm.js depends on 'lodash/f. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\core\lib\jsonforms-core.esm.js depends on 'lodash/ft'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\core\lib\jsonforms-core.esm.js depends on 'lodash/h CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\core\lib\jsonforms-core.esm.js depends on 'lodash/ides'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\core\lib\jsonforms-core.esm.js depends on 'lodash/ial'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\core\lib\jsonforms-core.esm.js depends on 'lodash/iction'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\core\lib\jsonforms-core.esm.js depends on 'lodash/k. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\core\lib\jsonforms-core.esm.js depends on 'lodash/r'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\core\lib\jsonforms-core.esm.js depends on 'lodash/re'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\core\lib\jsonforms-core.esm.js depends on 'lodash/remove'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\core\lib\jsonforms-core.esm.js depends on 'lodash/toPairs'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: <snip>\node_modules\@jsonforms\core\lib\jsonforms-core.esm.js depends on 'lodash/union'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
```
